app/controllers/graphql_controller.rb

```diff
@@ -15,6 +14,21 @@ def execute
   end
 
   private
+  def current_user
+    credential = setup_credentials
+
+    return unless credential
+    credential.current_user_id
+  end
+
+  def setup_credentials
```

Comment:
Copied your (@camallen) code pretty much directly for this. Is the match searching for a specific line in the JWT?